app/models/user.rb

```diff
@@ -529,13 +530,20 @@ def increase_daily_exports_counter!
   end
 
   def has_available_exports?
-    limit = (Rails.application.secrets.export_all_limit_24h || 3).to_i
-    last_export = export_vars[:last_export_timestamp]
+    last_export = export_vars[:last_export_timestamp] || 0
```

Comment:
I know this is a bit nitty gritty, but wouldn't it make more sense to name this variable `last_export_timestamp`. The way it is named now it implies we're dealing with whole export, not just timestamp.

Reply:
Or maybe last_export_at